### PR TITLE
Update CLI prompt's app run instructions.

### PR DESCRIPTION
**Pull request by rectified95**
Update CLI prompt message to be in line with new run commands.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/4416)

### CODE REVIEW

**Comment by NickGerleman** [comment]
You'll need to run `yarn change` to generate a changelog entry to make CI pass.

File: vnext/local-cli/generator-windows/index.js [hunk 1]
```diff
@@ -116,8 +116,8 @@ function copyProjectTemplateAndReplace(
   copyAndReplaceAll(path.join(srcPath, 'assets'), destPath, path.join(windowsDir, newProjectName, 'Assets'), templateVars, options.overwrite);
   copyAndReplaceAll(path.join(srcPath, 'src'), destPath, path.join(windowsDir, newProjectName), templateVars, options.overwrite);
 
-  console.log(chalk.white.bold('To run your app on UWP:'));
-  console.log(chalk.white('   react-native run-windows'));
+  console.log(chalk.white.bold('To run your app on Windows:'));
+  console.log(chalk.white('   npx react-native run-windows'));
 }
```
Comment (asklar) [inline vnext/local-cli/generator-windows/index.js]: this _shouldn't_ be necessary - when you install the react-native package globally it should create a .cmd file in a folder in your appdata that is in your PATH env var.
Do you run into this if you npm install **-g** react-native ?

Comment (acoates-ms) [reply vnext/local-cli/generator-windows/index.js]: Installing react-native globally (or most npm packages) is no longer recommended.  It gets out of date, and there is nothing keeping the version you have globally in sync with what you have in your local package.  It is also impossible to have multiple react-native apps on different versions if you are relying on global install state.

Comment (asklar) [reply vnext/local-cli/generator-windows/index.js]: good to know :) we should update the docs to reflect this

**Pull request closed without merging**